src/server/status_server/mod.rs

```diff
@@ -734,6 +741,7 @@ where
                                 Self::handle_resume_grpc(grpc_service_mgr)
                             }
                             (Method::GET, "/async_tasks") => Self::dump_async_trace(),
+                            (Method::GET, "/cached_regions") => Self::handle_dumple_cached_regions(in_memory_engine.as_ref()),
```

Comment:
Would you mind placing it in `debug/ime/cached_regions`? I'm thinking of adding a `debug/ime/dump_region` API to dump the metadata and data of a specific cached region. I think it's clearer to put them under the `debug/ime` prefix to indicate their scope.

Reply:
Done.